OrbitCore/TimerManager.cpp

```diff
@@ -137,17 +137,9 @@ void TimerManager::ConsumeTimers() {
       --m_NumQueuedEntries;
       --m_NumQueuedTimers;
 
-      // if( Timer.m_SessionID == Message::GSessionID ) // TODO: re-enable
```

Comment:
(Just for my understanding.) Ok, this is not enabled here as we check on a message level and not per timer.

Reply:
Yes exactly.  This is where the filtering happened on Windows, but on Linux we have more messages that we need to filter, so the check is moved upstream.  It should be fine on Windows as well btw.